packages/schematics/angular/guard/files/__name@dasherize__.guard.ts

```diff
@@ -1,14 +1,24 @@
 import { Injectable } from '@angular/core';
-import { CanActivate, ActivatedRouteSnapshot, RouterStateSnapshot } from '@angular/router';
+import { <%= implementationImports %>ActivatedRouteSnapshot, RouterStateSnapshot } from '@angular/router';
 import { Observable } from 'rxjs';
 
 @Injectable({
   providedIn: 'root'
 })
-export class <%= classify(name) %>Guard implements CanActivate {
-  canActivate(
+export class <%= classify(name) %>Guard implements <%= implementations %> {
+  <% if (implements && implements.includes('CanActivate')) { %>canActivate(
     next: ActivatedRouteSnapshot,
     state: RouterStateSnapshot): Observable<boolean> | Promise<boolean> | boolean {
     return true;
   }
+  <% } %><% if (implements && implements.includes('CanActivateChild')) { %>canActivateChild(
+    next: ActivatedRouteSnapshot,
+    state: RouterStateSnapshot): Observable<boolean> | Promise<boolean> | boolean {
+    return true;
+  }
+  <% } %><% if (implements && implements.includes('CanLoad')) { %>canLoad(
```

Comment:
nit: can now remove the `implements` truthy check here and above.

Reply:
Okay pushed those suggestions!